.travis.yml

```diff
@@ -2,6 +2,11 @@ language: python
 python:
   - 3.6
 dist: xenial
+
+env:
+  global:
+    - NODE_VERSION=dubnium
```

Comment:
I had to look up what dubnium was. I'd probably prefer a version number here, or at least a comment to clarify I suppose. We reference `10.16` in the docs below. It might make most sense to just pin a release version here.

Reply:
Changed to use the latest 10.x serie, 10.17.0